.swift-version

```diff
@@ -1 +1 @@
-2.3
+3.1
```

Comment:
It should have been, yeah. I'm kind of surprised CocoaPods didn't shout about it during pod install. I think if it didn't cause a problem for the release builds then maybe we should leave it, unless you guy have strong feelings about it.